tests/phpunit/Integration/I18nJsonFileIntegrityTest.php

```diff
@@ -5,7 +5,7 @@
 use SMW\Tests\Utils\UtilityFactory;
 
 /**
- * @group semantic-compound-queries
```

Comment:
This relates to PHPUnit https://phpunit.de/manual/4.0/en/appendixes.annotations.html#appendixes.annotations.group.